shell/platform/android/io/flutter/embedding/engine/FlutterEngineGroup.java

```diff
@@ -188,6 +217,9 @@ public static class Options {
     @Nullable private DartEntrypoint dartEntrypoint;
     @Nullable private String initialRoute;
     @Nullable private List<String> dartEntrypointArgs;
+    @NonNull PlatformViewsController platformViewsController = new PlatformViewsController();
```

Comment:
Please add "private" to fields here and below.

Reply:
Yeah, I made a mistake

Reply:
Done